app/assets/js/script.js

```diff
@@ -4,6 +4,11 @@
   var $win = $(window);
   var $doc = $(document);
 
+  // For assetgraph to pick up strings as urls
```

Comment:
Could it be done in a comment or something else that gets stripped out in the final build so there isn't JS specifically for the build system in the outputted code?